http/http-advanced-reactive/src/test/java/io/quarkus/ts/http/advanced/reactive/CookiesIT.java

```diff
@@ -52,6 +53,26 @@ void testSameSiteAttributeAddedByVertxHttpExt() {
                 .cookie("vertx", detailedCookie().sameSite(sameSite).secured(true));
     }
 
+    @Test
+    void testCookiesSerialization() {
+        given()
+                .get("/cookie/newcookie-serialization")
+                .then()
+                .statusCode(200)
+                .body(stringContainsInOrder(String.format("\"name\":\"%s\"", TEST_COOKIE),
+                        String.format("\"value\":\"%s\"", TEST_COOKIE_VALUE)));
+    }
+
+    @Test
+    void testCookiesSerialization3() {
+        given().cookie(String.format("%s=\"%s\";", TEST_COOKIE, TEST_COOKIE_VALUE))
+                .get("/cookie/cookie-serialization")
+                .then()
+                .statusCode(200)
+                .body(stringContainsInOrder(String.format("\"name\":\"%s\"", TEST_COOKIE),
```

Comment:
`stringContainsInOrder` expects cookies in some order. I don't know by which implementation is `httpHeaders.getCookies()` map backed, but can we really be sure it will always be in same order? either return some sorted map so that we can be sure, or use `containsInAnyOrder`

Reply:
It was not checking more cookies but values in one cookie. But after thinking about it. Even position of cookie value can be moved so replaced by `contains` as the `containsInAnyOrder` expected iterable not single string.